packages/e2e/test/wallet/PersonalWallet/multiAddress.test.ts

```diff
@@ -92,10 +93,17 @@ describe('PersonalWallet/multiAddress', () => {
 
     await walletReady(newWallet.wallet);
     const walletAddresses = await firstValueFromTimed(newWallet.wallet.addresses$);
+    const rewardAddresses = await firstValueFromTimed(newWallet.wallet.delegation.rewardAccounts$);
 
     // Let's check if all addresses has been discovered.
     expect(walletAddresses).toEqual(addressesToBeDiscovered);
 
+    // All addresses are built using the same staking key. Check that there is a single reward account
```

Comment:
```suggestion
    // All addresses are built using the same stake key. Check that there is a single reward account
```

Reply:
Updated [here](https://github.com/input-output-hk/cardano-js-sdk/pull/766/commits/93fbd73e1f7951220c8589f5ea4bdd098d903a02#diff-2326942f8aa170d248f8e7de0c3dbc2b452eba4354ab7dd781038656c968c179R101).
I force pushed since there was only a single comment.